src/lib/server/orders.ts

```diff
@@ -1048,11 +1047,11 @@ export async function addOrderPayment(
 	 */
 	opts?: { expiresAt?: Date | null; session?: ClientSession }
 ) {
-	if (order.status !== 'pending') {
```

Comment:
I think you can remove this line and set the order to `pending` initially like with non-free apyment method?

And remove the call to `updateAfterOrderPaid` as well in `createOrder` in the case of free payment method

Reply:
Marking unresolved, this is not done:

- set the order to pending initially like with non-free apyment method?
- remove the call to updateAfterOrderPaid as well in createOrder in the case of free payment method

Reply:
> remove the call to updateAfterOrderPaid as well in createOrder in the case of free payment method

already removed ?